### ISSUE TITLE
A6. Mathematics

### ISSUE THREAD

**Issue by iherman** [w3c/dpub-accessibility#32]
I think this section should be amended in view of the newly created CG:

https://www.w3.org/community/mathonwebpages/

The group plans to address rendering issues _as well as_ accessibility related issues. This CG should be mentioned and, I believe, the right approach is for the DPUB IG A11y TF members to follow what is happening in that CG and, possibly, join the discussion.

Cc: @pkra

**Comment by pkra** [comment]
Assigned to myself.
